src/miner.cpp

```diff
@@ -793,7 +793,7 @@ namespace pos {
                     {
                         if (shouldIgnoreMint(ctxState.subNode, blockHeight, creationHeight, subNodesBlockTime, chainparams)) 
                             break;
-                        LogPrint(BCLog::STAKING, "MakeStake: kernel found\n");
+                        LogPrintf("MakeStake: kernel found\n");
```

Comment:
How about we leave this to staking log, but instead tweak the other (`LogPrint(BCLog::STAKING, "ThreadStaker: (%s) minted a block!\n", operatorName);`) one to always print since that's more informative?